indico/modules/rb_new/client/js/components/App.module.scss

```diff
@@ -96,7 +96,13 @@
     }
 }
 
-.rb-pushable, .rb-pusher:global(.pusher) {
+.rb-pushable {
     // make sure that side bar takes the whole remaining height of the page
     min-height: calc(100vh - #{$menu-height} - 50px);
+    transform: initial !important;
```

Comment:
A comment explaining the need for this would be nice. We won't be able to figure it out in a few months, otherwise.